### ISSUE TITLE
ｒｅｓｕｌｔ

### ISSUE THREAD

**Issue by stepByStepToSky** [FesianXu/SK-CNN#1]
ｈｅｌｌｏ， I wonder that is your results are the same as the results in the paper??

**Comment by FesianXu** [comment]
@stepByStepToSky
Hi, this paper evaluates on Berkeley MHAD and ChaLearn Gesture Recognition Dataset. However i just evaluate it on NTU RGB+D. Here is the detail:
I just roughly evaluate this model on NTU RGB+D without randomly sampling patches from corner and center, and reach to nearly 72% in cross_subject experiments without fine tuning and i think the result will be better if i totally follow the train and test protocol in the origin paper in the evaluation on NTU RGB+D(i didnt do it because i evaluate the total model on my own database and just evaluate the partial one on NTU for trying)
I am quite sure that i totally follow the processing as origin papers in my code. Maybe you could evaluate it on the Berkeley MHAD and ChaLearn Gesture Recognition Dataset.

**Comment by stepByStepToSky** [comment]
Thank you very much. I am also puzzled that how these body parts compose an image. In the paper, the author does not write details about how to arrange the order of the body parts. I wonder why you connect these body parts that shown in your code. Thanks for your help.

**Comment by FesianXu** [comment]
@stepByStepToSky 
yes, the author didnt specify the arrangement of the skeleton so i specify one according to the paper. But i am not sure whether it is the best or not. Maybe you could try another arrange strategy.

**Comment by stepByStepToSky** [comment]
yes, I try this code on the  ChaLearn Gesture Recognition Dataset,  I only get accuracy result 72%,  but the paper get 91%. I get confused.

**Comment by FesianXu** [comment]
@stepByStepToSky 
i use test data as validation data and set the random flip rate to 0.6 in training step.
Have you found any other difference in my code between the description in the origin paper?
btw, i have not released the data organizing formation yet(i will finished it later), i am not sure if you know how i organized the original data. you could take this as reference. https://github.com/FesianXu/PLSTM

**Comment by ahmedbhna** [comment]
Could you share your implementation in NTU-RGB-D dataset
